src/ray/object_manager/plasma/store.cc

```diff
@@ -554,8 +556,15 @@ void PlasmaStore::PrintAndRecordDebugDump() const {
 }
 
 void PlasmaStore::RecordMetrics() const {
-  // TODO(sang): Add metrics.
+  absl::MutexLock lock(&mutex_);
+  RAY_LOG(DEBUG) << "Exporting plasma store metrics.";
   object_lifecycle_mgr_.RecordMetrics();
+
+  // Schedule the next metric export callback
+  metrics_timer_ = execute_after(
+      io_context_,
+      [this]() { RecordMetrics(); },
+      RayConfig::instance().metrics_report_interval_ms() / 2);
```

Comment:
Can you comment why?

Reply:
I guess the metric sampling interval should at least be smaller/equal to the metric reporting interval?

Reply:
oh I just meant to comment why (I know why lol) haha